internal/config/validate.go

```diff
@@ -72,7 +72,17 @@ func (v ValidationResults) Error() string {
 	return fmt.Sprintf("%d validation errors: %s", len(v), strings.Join(values, ", "))
 }
 
-const AppeningHappening = "[DEPRECATION] More than one app stanza within a waypoint.hcl file is deprecated, and will be removed in 0.10.\nPlease see https://discuss.hashicorp.com/t/deprecating-projects-or-how-i-learned-to-love-apps/40888 for more information."
+func (v ValidationResults) Errors() bool {
+	for _, vr := range v {
+		if vr.Error != nil {
+			return true
+		}
+	}
+
+	return false
+}
+
+const AppeningHappening = "[DEPRECATION] More than one app stanza within a waypoint.hcl file is deprecated, and will be removed in a future version.\nPlease see https://discuss.hashicorp.com/t/deprecating-projects-or-how-i-learned-to-love-apps/40888 for more information."
```

Comment:
> [DEPRECATION] More than one app stanza within a waypoint.hcl file is deprecated, and will be removed in a future version

I thought we were still debating/researching this, perhaps we should be softer here and mention it's something we're thinking about but not 100% committed either way?

Reply:
nvm!